.github/workflows/sycl_mlir_weekly.yml

```diff
@@ -0,0 +1,89 @@
+    if: ${{ always() && !cancelled() && needs.ubuntu2204_build.outputs.build_conclusion == 'success' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Intel L0 GPU
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: ext_oneapi_level_zero:gpu
+            reset_gpu: true
+
+          - name: Intel OCL GPU
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:gpu
+            reset_gpu: true
+
+          - name: OCL CPU
+            runner: '["Linux", "x86-cpu"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001
+            target_devices: opencl:cpu
+
+    uses: ./.github/workflows/sycl_linux_run_tests.yml
+    with:
+      name: ${{ matrix.name }}
+      runner: ${{ matrix.runner }}
+      image: ${{ matrix.image }}
+      image_options: ${{ matrix.image_options }}
+      target_devices: ${{ matrix.target_devices }}
+      reset_gpu: ${{ matrix.reset_gpu }}
+      ref: ${{ github.sha }}
+      merge_ref: ''
+      sycl_toolchain_artifact: sycl_linux_default
+      sycl_toolchain_archive: ${{ needs.ubuntu2204_build.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.ubuntu2204_build.outputs.artifact_decompress_command }}
+
+  sycl-bench_mlir_build:
+    needs: [ubuntu2204_build]
+    uses: ./.github/workflows/sycl-bench_build_and_test.yml
+    with:
+      repository_owner: whitneywhtsang
+      ref: sycl-mlir-enable-benchs
+      build_cache_root: /__w/
+      build_sycl_impl: LLVM-MLIR
+      sycl_toolchain_artifact: sycl_linux_default
+      sycl_toolchain_archive: ${{ needs.ubuntu2204_build.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.ubuntu2204_build.outputs.artifact_decompress_command }}
+      artifact_archive_name: sycl-bench_mlir.tar.zst
+      build_configure_extra_args: -DSYCL_BENCH_MLIR_BENCHMARKS_ONLY=ON
+      build_artifact_suffix: mlir
+
+  sycl-bench_build:
+    needs: [ubuntu2204_build]
+    uses: ./.github/workflows/sycl-bench_build_and_test.yml
+    with:
+      repository_owner: whitneywhtsang
+      ref: sycl-mlir-enable-benchs
+      build_cache_root: /__w/
+      build_sycl_impl: LLVM
+      sycl_toolchain_artifact: sycl_linux_default
+      sycl_toolchain_archive: ${{ needs.ubuntu2204_build.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.ubuntu2204_build.outputs.artifact_decompress_command }}
+      artifact_archive_name: sycl-bench.tar.zst
+      build_configure_extra_args: -DSYCL_BENCH_MLIR_BENCHMARKS_ONLY=ON
+      build_artifact_suffix: dpcpp
```

Comment:
Can we use some other suffix to indicate that the experimental MLIR is somehow used here? I'd like to avoid confusion for anybody outside looking into those result that it is somehow achieved using our released products.

Reply:
The `dpcpp`-suffixed artifact will be compiled using "standard" DPC++. Only the `mlir`-suffixed one will be compiler using the SYCL-MLIR compiler.